src/webapp/README.md

```diff
@@ -0,0 +1,118 @@
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- An Azure Active Directory (Azure AD) tenant and required permissions to create App registrations.
+- Python 3.8+
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting | Value for this sample app                                                  | Notes                                                                                              |
+|:----------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                     | `python-webapp`                                                              | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**  | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**            | **Web**                                                                      | Required value for this sample. <br/> Enables the required and optional settings for the app type. |
+| **Redirect URI**             | `http://localhost:5000/auth/redirect`                                        | Required value for this sample.                                                                    |
+| **Client secret**            | _Value shown in Azure portal_                                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+| **Implicit grant & hybrid flows** | _None selected_                                                         | This sample does not use the implicit grant or hybrid flows.                                       |
+| **Allow public client flows** | **No**                                                                      | This sample does not use a public client flow.                                                     |
+| **Token configuration**      | _No additional claims_                                                       | This sample does not rely on any additional claims existing in the tokens.                         |
+| **App roles**                | _Add an App role called `admin` for use by **Users/Groups**_                 | Required value for this sample. <bn /> One route requires your user to be assigned this role.      |
```

Comment:
Shorten second column to just the UI element names, if possible.

Reply:
Not following.  The first column is the UI element names, the second column is the value for "this sample app."  Do you have an example that you can help me see what I should change?

Reply:
In short, avoid using any instructional wording in that column like the bolded ones here:

_**Add an** App role **called** `admin` **for use by** **Users/Groups**_

For app roles and possibly client secrets, we might need to use bullets in that column or even a separate table (especially if there are several roles to create). We need to be explicit for all required values, as well. For example, I also need to know what to enter for **Value** and **Description**:

![image](https://user-images.githubusercontent.com/13394541/146107774-969e6822-debd-4717-8c90-5ab6c42ee69a.png)

Reply:
Ah, that makes sense. Okay, tightened that slop up and added the additional fields in: https://github.com/Azure-Samples/ms-identity-docs-code-python/pull/4/commits/760ab64e273668380ed27b1b29c5051b87244d98

Reply:
Dig it.